index.ts

```diff
@@ -25,7 +22,7 @@ app.use('/user', userRouter);
 app.use('/students', studentRouter);
```

Comment:
Tu powinno być '/student' nie '/students'. Chyba, ze jest powód, wtedy uzasadnij skąd ta zamiana.

Reply:
Nie pamiętam, żebym to zmieniał, może literówka - poprawię, dzięki!